Makefile

```diff
@@ -0,0 +1,27 @@
+export PREFIX := /usr/
+
+SUBDIRS := utils/ data/default/
+
+.PHONY: all $(SUBDIRS)
+
+all: install $(SUBDIRS)
+
+$(SUBDIRS):
+	$(MAKE) -C $@
+
+.PHONY: install
+
+install: settings.sh
+	mkdir -p $(DESTDIR)$(PREFIX)share/instantsettings
+	install -Dm 755 settings.sh $(DESTDIR)$(PREFIX)bin/instantsettings
+	install -m 644 instantsettings.desktop $(DESTDIR)$(PREFIX)share/applications/
+	install -m 644 instantcontrolcenter.desktop $(DESTDIR)$(PREFIX)share/applications/
+
+.PHONY: uninstall
+
+uninstall:
+	rm -r $(DESTDIR)$(PREFIX)share/instantsettings
+	rm $(DESTDIR)$(PREFIX)bin/instantsettings
+	rm $(DESTDIR)$(PREFIX)share/applications/instantsettings.desktop
+	rm $(DESTDIR)$(PREFIX)share/applications/instantcontrolcenter.desktop
```

Comment:
If an earlier `rm` fall fails e.g. because the file was already delete, the whole rule will fail. Usually `rm -f` is used to circumvent that.